form.yml.erb

```diff
@@ -117,21 +128,24 @@ attributes:
           data-min-ppn-pitzer: 1,
           data-max-ppn-pitzer: 48,
           data-option-for-owens: true,
-          data-option-for-pitzer: true
+          data-option-for-pitzer: true,
+          data-option-for-kubernetes: false
```

Comment:
Why not allow `any` as an option for k8s?

Reply:
Because if we do `any` we don't know if we need to mount `/apps` or `/usr/local` and we can't mount both because we also have to mount the Lmod initialization script and that is very cluster specific.  Also the underlying hardware for each cluster labeled node is different so it's guaranteed that running on pitzer worker node will work to execute all things compiled into /apps but not necessarily the case a Owens node can run everything in Pitzer's /apps.

Reply:
:+1: cool